src/controllers/ordersController.js

```diff
@@ -2,14 +2,21 @@ const ordersService = require("../services/ordersService");
 
 const placeOrder = async (req, res) => {
   const userId = req.userId;
-  const orderNumber = await ordersService.placeOrder(userId);
+  const orderResult = await ordersService.placeOrder(userId);
 
-  res.status(200).json({
-    message: "Order placed successfully",
-    orderNumber: orderNumber
-  });
+  if (orderResult.error) {
+    res.status(400).json({
+      message: orderResult.error
+    });
+  } else {
+    res.status(200).json({
+      message: "Order placed successfully",
+      orderNumber: orderResult
+    });
+  }
```

Comment:
에러가 났을 때 에러를 return하게 되면 else 없이 코드를 진행 할 수 있습니다.
```suggestion
  if (orderResult.error) {
   return res.status(400).json({
      message: orderResult.error
    });
  } 
  res.status(200).json({
    message: "Order placed successfully",
    orderNumber: orderResult
  });
  
```